platform/security/sts/security-sts-ldapclaimshandler/src/main/java/ddf/security/sts/claimsHandler/RoleClaimsHandler.java

```diff
@@ -207,7 +215,34 @@ public ProcessedClaimCollection retrieveClaimValues(ClaimCollection claims,
 
             connection = connectionFactory.getConnection();
             if (connection != null) {
-                BindResult bindResult = connection.bind(bindUserDN, bindUserCredentials.toCharArray());
+                BindRequest request;
+                switch (bindMethod) {
+                case "Simple":
```

Comment:
No reason that I can think of, that is just how I did it in this patch. If people want to change it, that is totally fine with me.